fastapi/params.py

```diff
@@ -316,6 +317,12 @@ def __init__(self, dependency: Callable = None, *, use_cache: bool = True):
         self.dependency = dependency
         self.use_cache = use_cache
 
+    def __repr__(self) -> str:
+        attr = getattr(self.dependency, "__name__",
+                       type(self.dependency).__name__)
+
+        cache = '' if self.use_cache else ',use_cache=False'
```

Comment:
I don't know how important is it, but why not adding the `use_cache=True` as well?